### PR TITLE
Selenium

### PR DESCRIPTION
**Pull request by WEremite**
Implement searching user's query on google.com with Selenim and saving result to file or print them in console.

### CODE REVIEW

File: search.py [hunk 1]
```diff
@@ -1,67 +1,73 @@
...
 
 
 @click.command()
 @click.option('-s', '--search', "user_query", default='World', help="User's search query")
-@click.option('-b', '--browser', "browser", default='chrome', help="Browser to use")
+@click.option('-b', '--browser', "browser", default='Chrome', help="Browser to use")
 @click.option('-o', '--output', "output", default='not', help="File to save the results")
 def main(user_query, browser, output):
     print(f"Searching for {user_query} in {browser}...")
+    if browser.capitalize() == 'Firefox':
```
Comment (nikkrichko) [inline search.py]: great solution with capitalize

Comment (nikkrichko) [inline search.py]: extract logic of the opening browser to other class

File: search.py [hunk 2]
```diff
@@ -1,67 +1,73 @@
...
-    number_of_search_results = 10
-    url = f"https://www.google.com/search?q={replace_whitespace}&num={number_of_search_results}"
-    webbrowser.get(browser).open(url)
-    requests_result = requests.get(url)
-    soup_link = BeautifulSoup(requests_result.content, "html.parser")
-    links = soup_link.find_all("a")
+    driver.get("https://www.google.com")
 
-    my_file = Path(output)
+    search_bar = driver.find_element(by="name", value="q")
```
Comment (nikkrichko) [inline search.py]: extract logic of working to another class method

File: search.py [hunk 3]
```diff
@@ -1,67 +1,73 @@
...
+            links.append(link['href'])
+            titles.append(title)
 
-def to_console(links):
-    result = open('result.csv', "w")
-    result.write("Title,URL\n")
+    my_file = Path(output)
 
-    add_results(links, result)
+    if my_file.exists():
```
Comment (nikkrichko) [inline search.py]: what would happen if the folder does not exist?

**Review by nikkrichko — CHANGES_REQUESTED** [review]

File: search.py (hunk 3 above)
Comment (WEremite) [reply search.py]: If folder or file does not exist or if there a typo - result will be shown in terminal
image

File: search.py (hunk 1 above)
Comment (WEremite) [reply search.py]: Done.

File: search.py (hunk 2 above)
Comment (WEremite) [reply search.py]: Done.

**Review by nikkrichko — APPROVED** [review]

**Pull request merged**